### PR TITLE
Documentation for the synchronous print request

### PR DESCRIPTION
**Pull request by sebr72**

### CODE REVIEW

File: docs/src/main/resources/templates/api.html [hunk 1]
```diff
@@ -289,6 +290,36 @@ <h4>Response</h4>
...
+  <p>
+    <code>:format</code> should be one of the formats supported by the specified print configuration (e.g.
+    <code>pdf</code> or <code>png</code>).
+  </p>
+
+  <h4>Response</h4>
+  <p>Status: <code>200 OK</code></p>
+
+  <h4>Request Sample</h4>
+  <p>Request URI: <code>DELETE /cancel/15179fee-618d-4356-8114-cfd8f146e273</code></p>
```
Comment (sbrunner) [inline docs/src/main/resources/templates/api.html:321]:
```suggestion
  <p>See <a href="#create">Create a print job</a> request sample</p>
```

**Review by sbrunner — APPROVED** [review]
Excellent, just one remark in the code :-)

File: docs/src/main/resources/templates/api.html (hunk 1 above)
Comment (sebr72) [reply docs/src/main/resources/templates/api.html:321]: Thanks. I don't know what happened with the pre-commit. But it is not the only problem, it got shifted in the middle of delete. I'll fix them all. Hopefully.

File: docs/src/main/resources/templates/api.html [hunk 2]
```diff
@@ -289,6 +290,37 @@ <h4>Response</h4>
...
+  <p>
+    <code>:format</code> should be one of the formats supported by the specified print configuration (e.g.
+    <code>pdf</code> or <code>png</code>).
+  </p>
+
+  <h4>Response</h4>
+  <p>Status: <code>200 OK</code></p>
+
+  <h4>Request Sample</h4>
+  <p>Request URI: <code>Request URI: POST /simple/buildreport.pdf</code></p>
```
Comment (sbrunner) [inline docs/src/main/resources/templates/api.html:321]: To respect the other syntax :-)
```suggestion
  <h4>Response Sample</h4>
  <p>Request URI: <code>POST /simple/buildreport.pdf</code></p>
```

Comment (sebr72) [reply docs/src/main/resources/templates/api.html]: Well spotted. There was no consistency in the other syntax. So I'll try to fix that.

**Review by sbrunner — APPROVED** [review]

**Pull request merged**